components/scilla/stdlib/BoolUtils.scilla

```diff
@@ -0,0 +1,30 @@
+library BoolUtils
```

Comment:
How come the Scilla `stdlib` is part of the assets here? I think it's better to leave this bundled together with the interpreter binaries.